OrbitLinuxTracing/include/OrbitLinuxTracing/Tracer.h

```diff
@@ -76,6 +81,9 @@ class Tracer {
 
   bool trace_context_switches_ = true;
   bool trace_callstacks_ = true;
+  // if trace_callstacks_ and trace_fp_callstacks_, use frame pointers for
```

Comment:
This comment is further evidence that we should prefer an enum over the double boolean for the callstack options.